static/css/cf-enhancements.less

```diff
@@ -146,7 +148,7 @@
     - cfgov-cf-enhancements
 */
 
-.respond-to-retina(@rules) {
+.respond-to-dpi(@rules) {
     @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```

Comment:
My suggestion on the PR is to pass the pixel ratio as a param (2 or 3) and change the media query based on that so we can support @3x devices with this mixin.